xml/System.Windows.Xps.Packaging/IXpsFixedPageReader.xml

```diff
@@ -1,439 +1,441 @@
+        <AssemblyVersion>3.0.0.0</AssemblyVersion>
+        <AssemblyVersion>4.0.0.0</AssemblyVersion>
+        <AssemblyVersion>5.0.0.0</AssemblyVersion>
+        <AssemblyVersion>6.0.0.0</AssemblyVersion>
+      </AssemblyInfo>
+      <ReturnValue>
+        <ReturnType>System.Int32</ReturnType>
+      </ReturnValue>
+      <Docs>
+        <summary>Gets the page number.</summary>
+        <value>An <see cref="T:System.Int32" /> that represents the page number.</value>
+        <remarks>To be added.</remarks>
+      </Docs>
+    </Member>
+    <Member MemberName="PrintTicket">
+      <MemberSignature Language="C#" Value="public System.Printing.PrintTicket PrintTicket { get; }" />
+      <MemberSignature Language="ILAsm" Value=".property instance class System.Printing.PrintTicket PrintTicket" />
+      <MemberSignature Language="DocId" Value="P:System.Windows.Xps.Packaging.IXpsFixedPageReader.PrintTicket" />
+      <MemberSignature Language="VB.NET" Value="Public ReadOnly Property PrintTicket As PrintTicket" />
+      <MemberSignature Language="F#" Value="member this.PrintTicket : System.Printing.PrintTicket" Usage="System.Windows.Xps.Packaging.IXpsFixedPageReader.PrintTicket" />
+      <MemberSignature Language="C++ CLI" Value="public:&#xA; property System::Printing::PrintTicket ^ PrintTicket { System::Printing::PrintTicket ^ get(); };" />
+      <MemberType>Property</MemberType>
+      <AssemblyInfo>
+        <AssemblyName>ReachFramework</AssemblyName>
+        <AssemblyVersion>3.0.0.0</AssemblyVersion>
+        <AssemblyVersion>4.0.0.0</AssemblyVersion>
+        <AssemblyVersion>5.0.0.0</AssemblyVersion>
+        <AssemblyVersion>6.0.0.0</AssemblyVersion>
+      </AssemblyInfo>
+      <ReturnValue>
+        <ReturnType>System.Printing.PrintTicket</ReturnType>
+      </ReturnValue>
+      <Docs>
+        <summary>Gets the <see cref="T:System.Printing.PrintTicket" />, if a ticket exists, that is associated with the <see cref="T:System.Windows.Documents.FixedPage" />.</summary>
+        <value>A <see cref="T:System.Printing.PrintTicket" /> that contains the default printing options for the page; or <see langword="null" /> if no <see cref="T:System.Printing.PrintTicket" /> exists for the page.</value>
+        <remarks>
+          <format type="text/markdown"><![CDATA[  
+  
+## Remarks  
+ For more information about the <xref:System.Printing.PrintTicket> part of an XML Paper Specification (XPS) page, see chapter 2 in the XPS specification, which you can obtain at [XPS: Specification and License Downloads](https://www.microsoft.com/whdc/xps/downloads.mspx).  
+  
+ ]]></format>
+        </remarks>
+        <altmember cref="N:System.Printing" />
+        <exception cref="F:System.Windows.Xps.SRID.ReachPackaging_PrintTicketAlreadyCommitted">part is <see langword="null" />.</exception>
+        <exception cref="F:System.Windows.Xps.SRID.ReachPackaging_NotAPrintTicket">part is <see langword="null" />.</exception>

```

Comment:
These descriptions are inconsistent compared to the other getters. Shouldn't this be:

```suggestion
        <exception cref="F:System.Windows.Xps.SRID.ReachPackaging_PrintTicketAlreadyCommitted">The print ticket has already been committed.</exception>
        <exception cref="F:System.Windows.Xps.SRID.ReachPackaging_NotAPrintTicket">Not a valid <see cref="T:System.Printing.PrintTicket"/> instance.</exception>

```